frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/disks/hotplug.spec.ts

```diff
@@ -0,0 +1,271 @@
+import { DISK_SOURCE, TEMPLATE_NAME, VM_ACTION_TIMEOUT, VM_STATUS, VM_ACTION } from '../../const';
+import { ProvisionSource } from '../../enums/provisionSource';
+import { testName } from '../../support';
+import { Disk, VirtualMachineData } from '../../types/vm';
+import { confirmButton, detailViewAction } from '../../view/actions';
+import { addDisk, delDisk, waitForCurrentVMStatus } from '../../view/dialog';
+import * as tags from '../../view/selector';
+import { tab } from '../../view/tab';
+import { virtualization } from '../../view/virtualization';
+import { vm, waitForStatus } from '../../view/vm';
+
+const vmData: VirtualMachineData = {
+  name: `test-vm-hotplug-${testName.split('-')[1]}`,
+  namespace: testName,
+  template: TEMPLATE_NAME,
+  provisionSource: ProvisionSource.URL,
+  pvcSize: '1',
+  sshEnable: false,
+  startOnCreation: true,
+};
+
+const hotPlugDisk: Disk = {
+  size: '2',
+};
+
+const FCB32_URL_IMG =
```

Comment:
We should use consts in `enums/provisionSource.ts` instead of creating new consts.

Reply:
I'll fix it